app/crud/base.py

```diff
@@ -303,7 +319,34 @@ def get_multi(
             A list of SQLAlchemy model instances from the query.
         """
         query = self.order_by(db.query(self.model), order_by=order_by)
-        return query.offset(skip).limit(limit).all()
+        data_from_db = query.offset(skip).limit(limit).all()
+        print(len(data_from_db))
+        count = query.count()
+        data = {
```

Comment:
`mypy` throws an error for this. It expects `PaginationSchemaType` for the returned value, but this is a `Dict` with an extra attribute (`results`), which does exist on `PaginationSchemaType`. One way to fix this is to make `PaginationSchemaType` generic, with `results` being `List[ModelType]`.

Reply:
Weird, because if you see, I have defined `PaginationSchemaType` to be of `TypeVar` bound to `BaseModel` and I pass in the Schema type present in the particular Schema. See line 12 of `crud_data_source.py` where I have passed in the schema of correct model type and that schema has a  field named as `results: List[DataSourceSummary]` and it inherits the `PaginationBase`. Also, I had some queries regarding this, could we discuss this over a quick call?

Reply:
Ah, I see. I'll test it again later. It might be just a matter of marking`data` with the correct type. Try running `mypy` or `./scripts/lint.sh` to see details of the error.

Reply:
It throws an error of unsorted imports :(

Reply:
Done, fixed it with `typing`'s `cast` function